tests/scripts/local_monitoring_tests/helpers/fixtures.py

```diff
@@ -0,0 +1,5 @@
+import pytest
```

Comment:
If helpers are common between local monitoring and remote monitoring, consider having a single copy

Reply:
helpers has common files which can be shared, except for - `generate_rm_jsons.py`, wherein both remote and local `target_cluster` need to be supported 
Will it be okay to add a new parameter in [create_exp_jsons()](https://github.com/shreyabiradar07/autotune/blob/mvp_demo/tests/scripts/remote_monitoring_tests/helpers/generate_rm_jsons.py#L34) to specify the target_cluster?
This way a single copy of helpers can be used

Reply:
Yes, but make sure the remote monitoring tests aren't affected

Reply:
[create_exp_jsons()](https://github.com/kruize/autotune/blob/mvp_demo/tests/scripts/remote_monitoring_tests/helpers/generate_rm_jsons.py#L34) is used by [kruize_pod_restart.py](https://github.com/kruize/autotune/blob/mvp_demo/tests/scripts/remote_monitoring_tests/fault_tolerant_tests/kruize_pod_restart_test.py#L91) (fault tolerant tests) and [test_list_recommendations.py](https://github.com/kruize/autotune/blob/mvp_demo/tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py#L459)
adding a new parameter to [create_exp_jsons()](https://github.com/kruize/autotune/blob/mvp_demo/tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py#L459) might affect remote monitoring tests

Reply:
You can add a default value to the parameter as remote and override it for local tests.